packages/tools/sandbox/src/components/dropUpButton.tsx

```diff
@@ -59,6 +57,8 @@ export class DropUpButton extends React.Component<IDropUpButtonProps, { isOpen:
             return null;
         }
 
+        const searchPlaceholder = this.props.searchPlaceholder ? this.props.searchPlaceholder : "Search...";

```

Comment:
nit: you could use `const searchPlaceholder = this.props.searchPlaceholder ?? "Search...";`

Reply:
done!